doc/api/v8.md

```diff
@@ -22,6 +22,7 @@ Returns an object with the following properties:
 * `total_available_size` {number}
 * `used_heap_size` {number}
 * `heap_size_limit` {number}
+* `does_zap_garbage` {bool}
```

Comment:
Ack, although it might be nice to have a short description here saying that this does return a 0/1 boolean.